docs/release_notes/9_5.md

```diff
@@ -96,7 +96,7 @@ Updated compiler toolsets:
 * **Rust Toolset 1.79.0**
 * **Go Toolset 1.22**
 
-Updated default Java implementation to `java-17-openjdk`:  
+Updated default Java implementation to `java-17-openjdk`:<br/>
```

Comment:
I'm sorry but double space is the mkdocs/markdown recommended way to make persistent newlines, using HTML in markdown is actively discouraged by linters.